### PR TITLE
browser: use safe-buffer WIP

### PR DESCRIPTION
**Pull request by dcousens**
- [x] Needs to be added to dependencies

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -23,8 +23,9 @@ function randomBytes (size, cb) {
   if (size > 0) {  // getRandomValues fails on IE if size == 0
     crypto.getRandomValues(rawBytes)
   }
-  // phantomjs doesn't like a buffer being passed here
-  var bytes = new Buffer(rawBytes.buffer)
+  
+  // XXX: phantomjs doesn't like a buffer being passed here
+  var bytes = Buffer.from(rawBytes.buffer)
```
Comment (calvinmetcalf) [inline browser.js]: you need to require safe buffer

**Comment by dcousens** [comment]
@calvinmetcalf I know,  this was a WIP,  I had hoped that was clear from the check box... my bad

File: browser.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ function oldBrowser () {
   throw new Error('secure random number generation not supported by this browser\nuse chrome, FireFox or Internet Explorer 11')
 }
 
+var Buffer = require('safe-buffer')
```
Comment (dcousens) [inline browser.js]: Fixed to be `require('safe-buffer')` in https://github.com/crypto-browserify/randombytes/commit/1870ca394623e1c88b8188a45a92a2dd7b0d5c53

**Comment by dcousens** [comment]
@calvinmetcalf I didn't realise `npm test` doesn't run `zuul` when merging. 
Waiting on Travis pass before merge.

**Comment by dcousens** [comment]
Published `v2.0.4`

**Pull request merged**